tests/pytests/functional/utils/user/test_get_group_dict.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import pytest
+
+import salt.utils.platform
+import salt.utils.user
+from tests.support.mock import patch
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.skip_on_windows,
+]
+
+
+@pytest.mark.skip_on_platforms(
+    darwin=True,
+    freebsd=True,
+    reason="This test should not run on FreeBSD and Mac due to lack of duplicate GID support",
+)
```

Comment:
```suggestion
pytestmark = [
    pytest.mark.skip_unless_on_linux(reason="Should only run in platforms which have duplicate GID support"),
]
```

Reply:
Thanks, it makes sense to do it this way :+1: